blt/blt.yml

```diff
@@ -231,6 +231,16 @@ validate:
       - files.php.tests
 
 tests:
+  cypress:
+    basePath: ${repo.root}/tests/cypress
+    paths:
+      - specDir: cypress/e2e/pretest/**/*.feature
```

Comment:
This gives me slight bad vibes. However, as long as there is just 1 feature this is ok. So maybe make a note here to be REAL careful.

Reply:
Yeah, it gave me bad vibes too. That's why I had originally wanted to do that as a discrete step.  We may have to revisit it one day, for now, I'll add a comment as we discussed.